src/requestgenerator.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs')
```

Comment:
I'd rename the file to match other .js file naming, probably `Request.js` as it's a class